models/how-to.md

```diff
@@ -0,0 +1,5 @@
+You find the official two and more existing models listed in the wiki:
```

Comment:
>You find the official two and more existing models listed in the wiki: 
I suggest to change this to a form: 
>You can find the a list of available pre-trained models for ocropy here: